sparse_ho/criterion.py

```diff
@@ -306,3 +346,80 @@ def get_val_grad(
             grad = None
 
         return val, grad
+
+
+class CrossVal():
```

Comment:
class docstring missing

Reply:
see on this rendered page https://qb3.github.io/sparse-ho/api.html that all classes and function docstrings are properly formated